packages/kotti-ui/source/kotti-popover/KtPopoverIconTextItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div
+		class="kt-popover-item"
+		:class="{
+			'kt-popover-item--is-disabled': isDisabled,
+		}"
+		v-on="$listeners"
+	>
+		<i v-show="icon" class="yoco" v-text="icon" />
+		<div>{{ label }}</div>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'KtPopoverIconTextItem',
+	props: {
+		icon: {
+			type: String,
+			default: null,
+		},
+		isDisabled: {
```

Comment:
so your rule of thumb for props could be that you either support the combo of [`type` and `default`] or [`type` and `required`]
so add a fallback `default` since you can't make this `required`